wled00/bus_manager.h

```diff
@@ -129,14 +132,14 @@ class Bus {
     }
     virtual bool hasWhite() { return Bus::hasWhite(_type); }
     static  bool hasWhite(uint8_t type) {
-      if ((type >= TYPE_WS2812_1CH && type <= TYPE_WS2812_WWA) || type == TYPE_SK6812_RGBW || type == TYPE_TM1814) return true; // digital types with white channel
+      if ((type >= TYPE_FW1906 && type <= TYPE_WS2812_WWA) || type == TYPE_SK6812_RGBW || type == TYPE_TM1814) return true; // digital types with white channel
```

Comment:
Why did you remove 1 channel WS2812?

Reply:
I do not remove 1 channel WS2812 I added FW1906. :)

Reply:
Sorry, I had in mind updated type to 30.